ns1/__init__.py

```diff
@@ -505,14 +506,15 @@ def createScope(
         dhcp_options=None,
         callback=None,
         errback=None,
+        **kwargs
```

Comment:
Ditto, if this will accept kwargs I think we want a docstring on here pointing out where the list of valid keywords is.